--- pritunl/auth/administrator.py	2017-01-14 19:39:27.000000000 +0800
+++ pritunl/auth/administrator.py	2017-01-12 04:43:51.000000000 +0800
@@ -332,7 +332,8 @@
 
         auth_string = '&'.join([
             auth_token, auth_timestamp, auth_nonce, flask.request.method,
-            flask.request.path])
+            flask.request.path] +
+            ([flask.request.data] if flask.request.data else []))
 
         if len(auth_string) > AUTH_SIG_STRING_MAX_LEN or len(auth_nonce) < 8:
             return False
